src/app/docs/toolbar/toolbar-docs.component.html

```diff
@@ -0,0 +1,147 @@
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Tile toolbar"
+      >
+        <p>
+          The tile toolbar appears in a <a href="https://developer.blackbaud.com/skyux/components/tiles">tile</a> above the control it affects. It contains actions to manipulate the contents of the tile or a portion of the tile. Only place the most important actions directly in the toolbar because tiles have limited horizontal real estate. Place all other actions in a more actions menu. If a tile requires no more than two filters, the filters should be docked on the right side of the toolbar.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/toolbar/tile-toolbar-context.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Composition"
+    >
+      <sky-docs-design-guideline-thumbnail
+        imageSource="~/assets/img/guidelines/toolbar/anatomy.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+
+      <sky-docs-design-guideline
+        heading="Standard toolbar actions"
+      >
+        <p>
+          Toolbars won't always need every possible button, but when present, buttons should be in the expected locations. Standard actions should appear from left to right in the following order when present in a toolbar:
+        </p>
+
+        <sky-toolbar-guidelines></sky-toolbar-guidelines>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Responsiveness"
+    >
+      <sky-docs-design-guideline
+        heading="Toolbar action wrapping"
+      >
+        <p>
+          The actions in a toolbar (including the more actions button) shouldn't extend beyond 50% of the screen. If the actions would extend beyond 50% of the screen, the rightmost action should collapse into the more actions menu.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Don't use toolbars with too many visible actions"
+          captionType="danger"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-dont-without-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do move the rightmost action into the more actions menu."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/toolbar/wrapping-do-with-context-menu.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p>
+          If a toolbar doesn't include a more actions menu and the actions extend beyond 50% of the screen, create a more actions menu for the two rightmost actions.
```

Comment:
I'd change the this to: `If a toolbar doesn't include a more actions menu but standard actions extend beyond 50 percent of the container, create a more actions menu for the two rightmost actions.`

The guidance to only include 2 seems weird to me. I wonder if that's something that Adam and Todd need to revise.

Reply:
Following up again. Do you think these changes are worth making, or should we just resolve the conversation?